firefox_desktop/explores/crashes.explore.lkml

```diff
@@ -26,14 +18,240 @@ explore: crash_usage {
     sql_on: ${crash_usage.country_code} = ${countries.code} ;;
   }
 
-  sql_always_where: {% condition crash_usage.date %} TIMESTAMP(${crash_usage.submission_date}) {% endcondition %}
-    AND {% condition crash_usage.date %} TIMESTAMP(${crashes_daily.submission_date}) {% endcondition %}
-    AND {% condition crash_usage.sample_id %} ${crash_usage.sample_id_dim} {% endcondition %}
-    AND {% condition crash_usage.sample_id %} ${crashes_daily.sample_id} {% endcondition %};;
+  sql_always_where: ${crash_usage.submission_date} >= "2021-01-01" ;;
 
   always_filter: {
     filters: [
-      crash_usage.date: "28 days",
+      crash_usage.submission_date: "3 month",
+      crash_usage.ten_percent_sample: "Yes"
     ]
   }
+
+  aggregate_table: ten_percent_daily_rollups {
+    query: {
+      dimensions: [
+        submission_date,
+        version,
+        major_version,
+        minor_version,
+        os_name,
+        channel,
+      ]
+      measures: [
+        active_hours,
+        dau_count,
+        content_crash_rate,
+        content_crashes,
+        content_crash_incidence,
+        gpu_crash_rate,
+        gpu_crashes,
+        gpu_crash_incidence,
+        main_crash_rate,
+        main_crashes,
+        main_crash_incidence,
+        rdd_crash_rate,
+        rdd_crashes,
+        rdd_crash_incidence,
+        socket_crash_rate,
+        socket_crashes,
+        utility_crash_incidence,
+        utility_crash_rate,
+        utility_crashes,
+        vr_crash_rate,
+        vr_crashes,
+        vr_crash_incidence,
+      ]
+      filters: [
+        crash_usage.days_since_build_published: "<=56",
+        crash_usage.ten_percent_sample: "Yes",
+      ]
+    }
+    materialization: {
+      sql_trigger_value: SELECT MAX(submission_date) FROM `moz-fx-data-shared-prod`.telemetry_derived.clients_daily_joined_v1 ;;
+      increment_key: "submission_date"
+    }
+  }
+
+  aggregate_table: ten_percent_build_id_rollups {
+    query: {
+      dimensions: [
+        build_id,
+        build_id_as_int,
+        channel,
+        os_name,
+      ]
+      measures: [
+        active_hours,
+        content_crash_rate,
+        gpu_crash_rate,
+        main_crash_rate,
+        rdd_crash_rate,
+        socket_crash_rate,
+        utility_crash_rate,
+        vr_crash_rate,
+      ]
+      filters: [
+        crash_usage.submission_date: "3 month",
+        crash_usage.days_since_build_published: "<=56",
+        crash_usage.ten_percent_sample: "Yes",
+      ]
+    }
+
+    materialization: {
+      sql_trigger_value: SELECT MAX(submission_date) FROM `moz-fx-data-shared-prod`.telemetry_derived.clients_daily_joined_v1 ;;
```

Comment:
This seems pretty expensive for a trigger check (every 5m I think?). Could you use the information schema? Something like [this](https://github.com/mozilla/looker-spoke-default/blob/e1315853507fc1ac6e78d252d53dc8df5f5f322b/mozilla_vpn/explores/active_subscriptions.explore.lkml#L75-L83):

 ```suggestion
       sql_trigger_value: SELECT MAX(last_modified_time) FROM `moz-fx-data-shared-prod`.telemetry_derived.INFORMATION_SCHEMA.PARTITIONS WHERE table_name = "clients_daily_joined_v1";;
```

Reply:
Good call 👍